tests/dummy/app/pods/demo/template.hbs

```diff
@@ -1,2 +1,58 @@
 frost-modal testbed
-{{frost-modal}}
+{{#frost-modal
+  confirm=(action 'confirmHandler') as |slot|}}
```

Comment:
not exactly because we want this attr to go to the top level ember-remodal component, so its then accessible in it's own onConfirm action handler